dbms/src/Storages/KVStore/tests/gtest_kvstore.cpp

```diff
@@ -1542,7 +1544,9 @@ TEST_F(RegionKVStoreOldTest, RegionRange)
         catch (Exception & e)
         {
             const auto & res = e.message();
-            ASSERT_EQ(res, "void DB::RegionsRangeIndex::remove(const RegionRange &, RegionID): not found region_id=1");
+            std::regex msg_reg(
+                "void DB::RegionsRangeIndex::remove\\(const .*RegionRange &, .*RegionID\\): not found region_id=1");
```

Comment:
```suggestion
                ".*: not found region_id=1");
```
Replacing the whole `__PRETTERY_FUNCTION__` by regex match is better

Reply:
Then we can remove `__PRETTERY_FUNCTION__` directly?

Reply:
"not found region_id=xxx" does not contain sufficient info about where the exception is thrown. If we catch and log the exception without its stacktrack, it is hard to find out where the error message come from.

While in this unit test, we just wan to ensure the left part contain the correct region_id without caring about the function name